L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -135,30 +107,29 @@ def customiseSimL1EmulatorForPostLS1_nomenu(process):
                 getattr(process, b).etHadSource = cms.InputTag("caloStage1LegacyFormatDigis")
                 getattr(process, b).hfRingEtSumsSource = cms.InputTag("caloStage1LegacyFormatDigis")
                 getattr(process, b).hfRingBitCountsSource = cms.InputTag("caloStage1LegacyFormatDigis")
-            #print "AFTER:  ", getattr(process, b).centralJetSource
-
-#    process.MessageLogger = cms.Service(
-#        "MessageLogger",
-#        destinations   = cms.untracked.vstring(
-#            'detailedInfo',
-#            'critical'
-#            ),
-#        detailedInfo   = cms.untracked.PSet(
-#            threshold  = cms.untracked.string('DEBUG')
-#            ),
-#        debugModules = cms.untracked.vstring(
-#            'l1tDigiToRaw', 'l1tRawToDigi'
-#            )
-#        )
-#    print process.HLTSchedule
+
+    return process
+
+
+from L1Trigger.Configuration.customise_overwriteL1Menu import *
+
+def customiseSimL1EmulatorForPostLS1_50ns(process):
+    # move to the 50ns v0 menu once the HLT has been updated accordingly
```

Comment:
Note that I'm still using the 25ns v2 L1 menu, as the HLT is not yet ready to use the dedicated 50ns v0 menu.